File: CS/DXBindingExample/ViewModel/BindingToMethodViewModel.cs
using DevExpress.Mvvm.DataAnnotations;

namespace DXBindingExample.ViewModel {
    [POCOViewModel]
    public class BindingToMethodViewModel : BaseViewModel {
        public override object Title { get { return "Binding To Function"; } }

        public int Sum(int x, int y) {
            return x + y;
        }
    }
}

File: VB/DXBindingExample/ViewModel/BindingToMethodViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations

Namespace DXBindingExample.ViewModel
    <POCOViewModel>
    Public Class BindingToMethodViewModel
        Inherits BaseViewModel

        Public Overrides ReadOnly Property Title() As Object
            Get
                Return "Binding To Function"
            End Get
        End Property

        Public Function Sum(ByVal x As Integer, ByVal y As Integer) As Integer
            Return x + y
        End Function
    End Class
End Namespace
